programs/util.h

```diff
@@ -127,6 +127,19 @@ extern int g_utilDisplayLevel;
     typedef struct stat stat_t;
 #endif
 
+int g_excludeCompressedFiles;
+static const char *g_compressedFileExtensions[] = {
```

Comment:
I believe a better design would be to create this list inside `fileio.c`,
and provide it as argument of the `UTIL_` function from `fileio.c`.

This would get rid of the circular dependency issue.

Reply:
Addressed in latest commit.